Wikipedia/Code/ArticleViewController+LinkPreviewing.swift

```diff
@@ -1,28 +1,151 @@
-extension ArticleViewController {
-    static let peekableImageExtensions: Set<String> = ["jpg", "jpeg", "gif", "png", "svg"]
-    
+
+protocol ArticleContextMenuPresenting {
```

Comment:
Since edit preview now leans on `ArticleContextMenuPreviewing`, it might be good to go ahead and move this declaration and the default implementations into a separate file. I did it [here](https://github.com/wikimedia/wikipedia-ios/commit/d6326c46b5bdcd508e98c0477c37c95e3b3affb7) if you feel like pulling it in.

Reply:
Was wondering if we had hit that point. Pulled in, thanks.